<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>160</width>
      <height>306</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Emergency">
       <property name="text">
        <string>Help! Emergency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SevenFloor">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SixthFloor">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FifthFloor">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fourthFloor">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ThirdFloor">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SecondFloor">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FirstFloor">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseDoorButton">
       <property name="text">
        <string>Close Door</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenDoorButton">
       <property name="text">
        <string>Open Door</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Elevator Simulation System</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>181</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Elevator Information</string>
    </property>
    <widget class="QLabel" name="elevatorLabel1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator1 Current Floor</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevatorLabel2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator2 Current Floor</string>
     </property>
    </widget>
    <widget class="QLabel" name="elevatorLabel3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Elevator3 Current Floor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FireEmergency">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Fire Emergency</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockDoorE1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Block Door E1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockDoorE2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Block Door E2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blockDoorE3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Block Door E3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unblockDoorE1">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unblockDoorE2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unblockDoorE3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="onBackupPower">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Backup Power</string>
     </property>
    </widget>
    <widget class="QPushButton" name="offBackupPower">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="powerOutage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Power Outage!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resolvePowerOutage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>82</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="upButton">
        <property name="text">
         <string>UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downButton">
        <property name="text">
         <string>DOWN</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QComboBox" name="currentFloorPassenger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="FireEmergencyOff">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
    <widget class="QComboBox" name="NumberOfPassenger">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="unOverloadE1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Unoverload E1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unOverloadE2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>E2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="unOverloadE3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>380</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>E3</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Elevator1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="elevatorDisplay1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor: 1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Elevator2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>261</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="elevatorDisplay2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor: 1</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>160</width>
       <height>306</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="Emergency_2">
        <property name="text">
         <string>Help! Emergency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SevenFloor_2">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SixthFloor_2">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FifthFloor_2">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fourthFloor_2">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ThirdFloor_2">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SecondFloor_2">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FirstFloor_2">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CloseDoorButton_2">
        <property name="text">
         <string>Close Door</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenDoorButton_2">
        <property name="text">
         <string>Open Door</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="Elevator3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>251</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="elevatorDisplay3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Floor: 1</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>160</width>
       <height>306</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QPushButton" name="Emergency3">
        <property name="text">
         <string>Help! Emergency</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SevenFloor3">
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SixthFloor3">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FifthFloor3">
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fourthFloor3">
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ThirdFloor3">
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SecondFloor3">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FirstFloor3">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CloseDoorButton3">
        <property name="text">
         <string>Close Door</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="OpenDoorButton3">
        <property name="text">
         <string>Open Door</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>Elevator1</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>groupBox</zorder>
   <zorder>Elevator2</zorder>
   <zorder>Elevator3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
